src/new-components/section-header/section-header.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled from 'styled-components';
+import { TextStyles } from '../../components/typography/typography-v2';
+import { up } from '../../components/breakpoint/breakpoint';
+
+interface BlogHeaderProps {
+  /**
+   * A short label above the headline
+   */
+  kicker?: string;
+  /**
+   * The headline for the given section
+   */
+  headline: string;
+  /**
+   * The actual text content
+   */
+  children?: React.ReactNode;
+}
+
+const KickerStyled = styled.span`
+  ${TextStyles.toplineR}
+  display: block;
+  color: #3e61ee;
+  margin-bottom: 16px;
+`;
+
+const HeadlineStyled = styled.h2`
+  ${TextStyles.headlineXL}
+  margin: 0;
+  color: #202840;
+  margin-bottom: 24px;
+
+  ${up('md')} {
+    margin-bottom: 32px;
+  }
+`;
+
+const ContentStyled = styled.div`
+  ${TextStyles.textR}
+
+  ${up('md')} {
+    ${TextStyles.textL}
+  }
+`;
+
+export const SectionHeader = (props: BlogHeaderProps) => {
+  return (
+    <div>
+      {props.kicker && <KickerStyled>{props.kicker}</KickerStyled>}
+      <HeadlineStyled>{props.headline}</HeadlineStyled>
+      <ContentStyled>{props.children}</ContentStyled>
```

Comment:
We need to have a naming convention for these styled components. It's `ContentStyled` in some places and `StyledContent` in others. I prefer using `Styled` as a prefix (e.g. `StyledContent`) because it feels more natural while reading ("styled content" vs. "content styled").
In the end I'm fine with both, but let's be consistent.

Reply:
it's just the rule I'm missing, my mind constantly goes around the available possibilities as I can't remember a single way 👍